modules/node_modules/@ncigdc/components/Aggregations/DateFacet.js

```diff
@@ -0,0 +1,142 @@
+/* @flow */
+
+import React from 'react';
+import { compose, withState, mapProps, pure } from 'recompose';
+import AngleIcon from 'react-icons/lib/fa/angle-down';
+import { SingleDatePicker } from 'react-dates';
+import 'react-dates/lib/css/_datepicker.css';
+import moment from 'moment';
+
+import './style.css';
+import { withTheme } from '@ncigdc/theme';
+
+import Input from '@ncigdc/uikit/Form/Input';
+
+import { Row, Column } from '@ncigdc/uikit/Flex';
+
+import Link from '../Links/Link';
+
+import type from './types';
+
+const enhance = compose(
+  withState('state', 'setState', {
+    collapsed: false,
+    date: moment(),
+    focused: false,
+  }),
+  mapProps(({ setState, ...rest }) => ({
+    toggleCollapsed: () => setState(state => ({ collapsed: !state.collapsed })),
+    handleDatePicked: (date) => setState(s => ({ ...s, date })),
+    setState,
+    ...rest,
+  })),
+  withTheme,
+  pure
+);
+
+type TProps = {
+  theme: Object,
+  field: string,
+  query: Object,
+  title: string,
+  state: {
+    collapsed: boolean,
+    date: any,
+    focused: boolean,
+  },
+  setState: Function,
+  style: Object,
+  toggleCollapsed: Function,
+  handleToChanged: Function,
+  handleFromChanged: Function,
+};
+
+const DateFacet = (props: TProps) => {
+  const styles = {
+    container: {
+      padding: '1rem 1.2rem',
+      backgroundColor: 'white',
+    },
+    header: {
+      color: props.theme.primaryLight1,
+      fontSize: '1.7rem',
+      marginBottom: '0.5rem',
+      cursor: 'pointer',
+      alignItems: 'center',
+    },
+    label: {
```

Comment:
yeah with the third party stuff it's fine. instead of importing a .css file you can do

```
import { insertRule } from 'glamor'
insertRule(`
  .stuff { color: red; }
`)
```

but I'm not sure that's better.. I do it just cause I don't want to leave js land hehe